templates/dotnet/Package/Models/Model.cs.twig

```diff
@@ -40,7 +40,7 @@ namespace {{ spec.title | caseUcfirst }}.Models
+            {{ property.name | caseCamel | escapeKeyword | removeDollarSign }}: {% if property.sub_schema %}{% if property.type == 'array' %}((JArray)map["{{ property.name }}"]).ToObject<List<Dictionary<string, object>>>().Select(it => {{property.sub_schema | caseUcfirst | overrideIdentifier}}.From(map: it)).ToList(){% else %}{{property.sub_schema | caseUcfirst | overrideIdentifier}}.From(map: ((JObject)map["{{ property.name }}"]).ToObject<Dictionary<string, object>>()!){% endif %}{% else %}{% if property.type == 'array' %}((JArray)map["{{ property.name }}"]).ToObject<{{ property | typeName }}>(){% else %}{% if property.type == "integer" or property.type == "number" %}{% if not property.required %}map["{{ property.name }}"] == null ? null : {% endif %}Convert.To{% if property.type == "integer" %}Int64{% else %}Double{% endif %}(map["{{ property.name }}"]){% else %}{% if property.type == "boolean" %}({{ property | typeName }}{% if not property.required %}?{% endif %})map["{{ property.name }}"]{% else %}map{% if not property.required %}.TryGetValue("{{ property.name }}", out var {{ property.name | caseCamel | escapeKeyword | removeDollarSign }}) ?  {{ property.name | caseCamel | escapeKeyword | removeDollarSign }}{% if property.name == "responseBody" or property.type | caseLower == "file" %} as Payload{% else %}?.ToString(){% endif %} : null{% else %}["{{ property.name }}"]{% if not property.required %}?{% endif %}{% if property.name == "responseBody" or property.type | caseLower == "payload" %} as Payload{% else %}.ToString(){% endif %}{% endif %}{% endif %}{% endif %}{% endif %}{% endif %}{% if not loop.last or (loop.last and definition.additionalProperties) %},{% endif %}
```

Comment:
```suggestion
{{ property.name | caseCamel | escapeKeyword | removeDollarSign }}:
{%~ if property.sub_schema %}
    {%~ if property.type == 'array' %}
        ((JArray)map["{{ property.name }}"])
            .ToObject<List<Dictionary<string, object>>>()
            .Select(it => {{property.sub_schema | caseUcfirst | overrideIdentifier}}.From(map: it))
            .ToList()
    {%~ else %}
        {{property.sub_schema | caseUcfirst | overrideIdentifier}}.From(
            map: ((JObject)map["{{ property.name }}"])
                .ToObject<Dictionary<string, object>>()!
        )
    {%~ endif %}
{%~ else %}
    {%~ if property.type == 'array' %}
        ((JArray)map["{{ property.name }}"])
            .ToObject<{{ property | typeName }}>()
    {%~ else %}
        {%~ if property.type == "integer" or property.type == "number" %}
            {%~ if not property.required -%}
                map["{{ property.name }}"] == null
                    ? null
                    : 
            {%~ endif -%}
            Convert.To{% if property.type == "integer" %}Int64{% else %}Double{% endif %}(map["{{ property.name }}"])
        {%~ else %}
            {%~ if property.type == "boolean" %}
                ({{ property | typeName }}{% if not property.required %}?{% endif %})map["{{ property.name }}"]
            {%~ else %}
                map{% if not property.required %}.TryGetValue("{{ property.name }}", out var {{ property.name | caseCamel | escapeKeyword | removeDollarSign }})
                    ? {{ property.name | caseCamel | escapeKeyword | removeDollarSign }}{% if property.name == "responseBody" or property.type | caseLower == "file" %} as Payload{% else %}?.ToString(){% endif %}
                    : null
                {%~ else %}
                    ["{{ property.name }}"]{% if not property.required %}?{% endif %}{% if property.name == "responseBody" or property.type | caseLower == "payload" %} as Payload{% else %}.ToString(){% endif %}
                {%~ endif %}
            {%~ endif %}
        {%~ endif %}
    {%~ endif %}
{%~ endif %}{% if not loop.last or (loop.last and definition.additionalProperties) %},{% endif -%}
```

Reply:
I like this but I think it should be done as part of code quality project panned later this week. We have issue for it.